Sources/Private/MainThread/LayerContainers/CompLayers/PreCompositionLayer.swift

```diff
@@ -69,6 +77,7 @@ final class PreCompositionLayer: CompositionLayer {
     childKeypaths.append(contentsOf: layers)
 
     layerImageProvider.addImageLayers(imageLayers)
+    layerTextProvider.addTextLayers(textLayers)
```

Comment:
Found and fixed an unrelated bug where the Main Thread rendering engine didn't correctly track and update text layers included in `PreCompositionLayer`s.